docs/_data/components/schemas/names.yaml

```diff
@@ -11,6 +11,8 @@ properties:
       properties:
         name:
           $ref: ./generic-identifier.yaml
+        type:
```

Comment:
I think that's fair.  I also don't think that we can force model authority- I understand your point about the value, but since CIM doesn't really force you to do it, I think this turns into a fight.  

I created a separate issue to track this.  Is that reasonable? https://github.com/trolie/spec/issues/174